prod.dockerfile

```diff
@@ -23,6 +23,9 @@ COPY packages/venia-concept/package.json ./packages/venia-concept/package.json
 COPY package.json yarn.lock babel.config.js magento-compatibility.js ./
 COPY scripts/monorepo-introduction.js ./scripts/monorepo-introduction.js
 
+# To handle "could not get uid/gid"
+RUN npm config set unsafe-perm true
+
```

Comment:
@zetlen On CI `yarn install` was failing with error https://console.aws.amazon.com/codesuite/codebuild/projects/pwa-pull-request-build/build/pwa-pull-request-build%3A4b8d13c6-fd9e-42d3-83a3-287a61eea4bd/log?region=us-east-1 .
I tried updating to latest docker images but ended up in one or other error.  You can check those errors in aws build logs for test pr https://github.com/magento/pwa-studio/pull/2385 commits.

I gave up on that and applied resolution from https://stackoverflow.com/questions/52196518/could-not-get-uid-gid-when-building-node-docker.